### PR TITLE
Added `spike_train_generation` module...

### PR DESCRIPTION
**Pull request by apdavison**
with functions to generate homogeneous Poisson and gamma processes.

### CODE REVIEW

File: doc/authors.rst [hunk 1]
```diff
@@ -4,11 +4,19 @@ Authors and contributors
 
 The following people have contributed code and/or ideas to the current version
 of Elephant. The institutional affiliations are those at the time of the
-contribution, sand may not be the current affiliation of a contributor.
+contribution, and may not be the current affiliation of a contributor.
 
 * Andrew Davison [1]
 * Pierre Yger [1]
+* Eilif Muller [2, 3]
+* Alper Yenenoglu [4]
```
Comment (alperyeg) [inline doc/authors.rst]: Can you correct my name to Yegenoglu ;)

Comment (apdavison) [inline doc/authors.rst]: Done. Sorry about that.

**Comment by apdavison** [comment]
@Vahidrostami - what do you think of my corrections?

**Comment by Vahidrostami** [comment]
@apdavison I went through your corrections. The code seems now good to me.

**Comment by apdavison** [comment]
@alperyeg or @mdenker  - please can you merge this now?

**Pull request merged**